package.json

```diff
@@ -43,24 +43,29 @@
   },
   "license": "MIT",
   "sideEffects": false,
+  "type": "module",
```

Comment:
Do we need to make this an ESM package? I think this should work without this, no?

Reply:
Reverted here: https://github.com/MetaMask/superstruct/pull/18/commits/6c4e6cdae9a38476d758036c664067e88bfe9232